cmd/gangway/handlers.go

```diff
@@ -94,12 +99,12 @@ func loginRequired(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		session, err := sessionStore.Get(r, "gangway")
 		if err != nil {
-			http.Redirect(w, r, "/", http.StatusTemporaryRedirect)
+			http.Redirect(w, r, fmt.Sprintf("%s", cfg.getRootPathPrefix()), http.StatusTemporaryRedirect)
```

Comment:
Maybe I'm overlooking something small but it seems like `getRootPathPrefix` already returns a `string`. What's the use case for the `fmt.Sprintf` ?

Reply:
ahh good catch! I refactored that and don't need the fmt any longer.